src/ResourceManager/Sql/Commands.Sql/help/Get-AzureRmSqlServerAuditingPolicy.md

```diff
@@ -10,6 +10,7 @@ schema: 2.0.0
 
 ## SYNOPSIS
 Gets the auditing policy of a SQL server.
+WARNING: The command **Get-AzSqlServerAuditingPolicy** is obsolete. Note that Table auditing is deprecated and this command will be removed in a future release. Please use the **Get-AzureRmSqlServerAuditing** command to get Blob auditing settings.
```

Comment:
@cormacpayne  These cmdlets are already deprecated, and we want to remove them in the next breaking-changes release.
Also, This command already produces a warning when executed (Added using the Obsolete attribute). Is it needed to add the warning in the description? If so, there are additional commands that are deprecated too and do not have this warning in their description.

Reply:
@yamatoya Hey Takahito, would you mind taking a look at the above comments? I believe that the warning messages produced from the cmdlets on execution should be sufficient to users, but let me know if you feel otherwise and we can take a look at where else we'll need to update the documentation to reflect this information.